kobo/apps/stripe/views.py

```diff
@@ -46,6 +44,96 @@ def get_queryset(self):
         ).prefetch_related('payment_intent')
 
 
+# Change an existing subscription to a new price
```

Comment:
I think we could add more documentation, either in the class docstring or on some static methods (or both). This class does a few things but it's not clear exactly what at a glance.